code/controllers/Processes/throwing.dm

```diff
@@ -0,0 +1,139 @@
+#define MAX_THROWING_DIST 512 // 2 z-levels on default width
+#define MAX_TICKS_TO_MAKE_UP 3 //how many missed ticks will we attempt to make up for this run.
+
+var/global/datum/controller/process/throwing/throw_master
+
+/datum/controller/process/throwing
+	var/list/processing_list
+
+/datum/controller/process/throwing/setup()
+	name = "throwing"
+	schedule_interval = 1
+	start_delay = 20
+	log_startup_progress("Throw ticker starting up.")
+	if(throw_master)
+		qdel(throw_master)
+	throw_master = src
+
+/datum/controller/process/throwing/statProcess()
+	..()
+	stat(null, "P:[processing_list.len]")
+
+/datum/controller/process/throwing/started()
+	..()
+	if(!processing_list)
+		processing_list = list()
+
+/datum/controller/process/throwing/doWork()
+	for(last_object in processing_list)
```

Comment:
This is definitely more like our standard controllers than space drifting, but, again, in the long run, I'm not sure it really matters.